BraintreeDropIn/BTCardFormViewController.m

```diff
@@ -274,6 +286,55 @@ - (void)updateRequiredFields {
     }
 }
 
+#pragma mark - card.io
+
+- (void)setupCardIO {
+    if ([self isCardIOAvailable]) {
+        BTUIKInputAccessoryToolbar *cardToolBar = [BTUIKInputAccessoryToolbar new];
+        UIBarButtonItem *flexSpace = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFlexibleSpace target:nil action:nil];        
+        UIBarButtonItem *cardIOButton = [[BTUIKBarButtonItem alloc] initWithTitle:BTUIKLocalizedString(SCAN_CARD_IO_ACTION) style:UIBarButtonItemStylePlain target:self action:@selector(presentCardIO)];
+        cardToolBar.items = @[flexSpace, cardIOButton, flexSpace];
+        self.cardNumberField.textField.inputAccessoryView = cardToolBar;
+    }
+}
+
+- (BOOL)isCardIOAvailable {
+    Class kCardIOView = NSClassFromString(@"CardIOPaymentViewController");
+    Class kCardIOUtilities = NSClassFromString(@"CardIOUtilities");
+    if (kCardIOView != nil && kCardIOView != nil
+        && [kCardIOUtilities respondsToSelector:@selector(libraryVersion)]
```

Comment:
I would use `cardIOLibraryVersion` especially if this method is called dynamically. https://github.com/card-io/card.io-iOS-source/blob/master/CardIO_Public_API/CardIOUtilities.h#L42 . Apple has flagged a few developers in app review where they used a method called `libraryVersion` thinking they were calling an internal private method.